docs/kubernetes/windows.md

```diff
@@ -1,446 +1,309 @@
-
-Affects: Some builds of Azure CNI
-
-Issues: https://github.com/Azure/azure-container-networking/issues/147
-
-Run `ipconfig /all` in a pod, and check that the first DNS server listed is within your cluster IP range (10.x.x.x). If it's not listed, or not the first in the list, then an azure-cni update is needed.
-
-Workaround:
-
-1.	Get the kube-dns service IP with `kubectl get svc -n kube-system kube-dns`
-2.  Cordon & drain the node
-3.	Modify `C:\k\azurecni\netconf\10-azure.conflist` and make it the first entry under Nameservers
-4.  Uncordon the node
-
-Example:
+```powershell
+PS C:\Users\patrick\acs-engine> .\kubectl.exe get pod
+NAME                        READY     STATUS              RESTARTS   AGE
+iis-1803-6c49777598-h45cs   0/1       ContainerCreating   0          1m
+PS C:\Users\patrick\acs-engine> .\kubectl.exe get service
+NAME         TYPE           CLUSTER-IP   EXTERNAL-IP   PORT(S)        AGE
+iis          LoadBalancer   10.0.9.47    <pending>     80:31240/TCP   1m
+kubernetes   ClusterIP      10.0.0.1     <none>        443/TCP        46m
 ```
-{
-    "cniVersion":  "0.3.0",
-    "name":  "azure",
-    "plugins":  [
-                    {
-                        "type":  "azure-vnet",
-                        "mode":  "tunnel",
-                        "bridge":  "azure0",
-                        "ipam":  {
-                                     "type":  "azure-vnet-ipam"
-                                 },
-                        "dns":  {
-                                    "Nameservers":  [
-                                                        "10.0.0.10",
-                                                        "168.63.129.16"
-                                                    ],
-                                    "Search":  [
-                                                   "default.svc.cluster.local"
-                                               ]
-                                },
-…
-```
-
-#### Pods cannot resolve cluster DNS names
-
-Affects: Azure CNI plugin <= 0.3.0
-
-Issues: https://github.com/Azure/azure-container-networking/issues/146
 
-If you can't resolve internal service names within the same namespace, run `ipconfig /all` in a pod, and check that the DNS Suffix Search List matches the form `<namespace>.svc.cluster.local`. An Azure CNI update is needed to set the right DNS suffix.
+Since this is the first deployment, it will probably take several minutes for the Windows node to download and run the container. Later deployments will be faster because the large `microsoft/windowsservercore` container will already be on disk. 
 
-Workaround:
-1.	Use the FQDN in DNS lookups such as `kubernetes.kube-system.svc.cluster.local`
-2.	Instead of DNS, use environment variables `* _SERVICE_HOST` and `*_SERVICE_PORT` to find service IPs and ports in the same namespace
+Once the pod is in `Running` state, get the IP from `kubectl get service` then visit `http://<external-ip>` to test your web server.

```

Comment:
explicitly call this external IP from the output above

Reply:
ok clarified it